src/main/scala/com/typesafe/sbt/packager/docker/dockerfile.scala

```diff
@@ -44,22 +44,38 @@ case class ExecCmd(cmd: String, args: String*) extends CmdLike {
  *
  * @example
  * {{{
- *   val add = Cmd("ADD", "src/resource/LICENSE.txt /opt/docker/LICENSE.txt")
+ *   val add = Cmd("ADD", "src/resource/LICENSE.txt", "/opt/docker/LICENSE.txt")
+ * }}}
+ * 
+ * @example
+ * {{{
+ *   val copy = Cmd("COPY", "src/resource/LICENSE.txt", "/opt/docker/LICENSE.txt")
+ * }}}
+ *
+ * @example
+ * {{{
+ *   val env = Cmd("ENV", "APP_SECRET", "7sdfy7s9hfisdufuusud")
  * }}}
  */
-case class Cmd(cmd: String, arg: String) extends CmdLike {
-  def makeContent = "%s %s\n" format (cmd, arg)
+case class Cmd(cmd: String, args: String*) extends CmdLike {
```

Comment:
I'm not 100% sure about this. `Cmd(cmd: String, first: String, rest: String*)` would represent the old interface more correctly. However with your suggestion zero-argument commands would be possible.

@jsuereth  what about binary  compatibility?